### PR TITLE
test: Fix ami marketplace pipeline

### PR DESCRIPTION
**Pull request by quentinDERORY**
## Summary

Add propagation of agw version to the ci pipeline
Fix some minor bugs

## Test Plan

https://app.circleci.com/pipelines/github/magma/magma/27773/workflows/3de72b04-e985-4c2c-b2ff-d4ed371d6bee/jobs/305603

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: experimental/cloudstrapper/playbooks/roles/magma-agw/tasks/agw-setup.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ---
 
 - name: run magma deploy
-  command: ansible-playbook -e '@{{ usrHome }}/ansible-env.yaml' -e "magma_pkgrepo_dist=focal-{{ buildAgwPackage }}"  -i {{ usrHome }}/inventory {{ deployPath }}/magma_deploy.yml
+  command: ansible-playbook -e "magma_pkgrepo_dist=focal-{{ buildAgwPackage }}"  -i {{ usrHome }}/inventory {{ deployPath }}/magma_deploy.yml
```
Comment (arunuke) [inline experimental/cloudstrapper/playbooks/roles/magma-agw/tasks/agw-setup.yaml:4]: No need for the variables from ansible-env anymore? Are they being preset elsewhere?

Comment (quentinDERORY) [reply experimental/cloudstrapper/playbooks/roles/magma-agw/tasks/agw-setup.yaml:4]: you deleted them in the previous PR, it's running fine without.

Comment (quentinDERORY) [reply experimental/cloudstrapper/playbooks/roles/magma-agw/tasks/agw-setup.yaml:4]: I just checked, they were not even used in the role itself.

Comment (arunuke) [reply experimental/cloudstrapper/playbooks/roles/magma-agw/tasks/agw-setup.yaml:4]: I am looking at https://github.com/arunuke/magma/blob/f04e4ec03d794d756213b74134d2925d707524c5/experimental/cloudstrapper/playbooks/roles/magma-agw/tasks/agw-setup.yaml (which is the one where I added buildAgwPackage to agw-setup) and it still has the ansible-env.yaml file there. That said, if the build works fine even without that file, then that's great.

**Review by arunuke — APPROVED** [review]
If build tests are passing, LGTM.

**Pull request merged**